tensorboard/plugins/profile/README.md

```diff
@@ -88,10 +88,11 @@ below.
 
 ### 2. JSON format for the Input-pipeline Analyzer
 
-JSON for the Input-pipeline Analyzer uses two separate DataTables: the first
+JSON for the Input-pipeline Analyzer uses three separate DataTables: the first
 DataTable contains statistics for device step time; the second DataTable
-contains statistics for host side state. Each DataTable is associated with some
-table properties and some tabular data.
+contains statistics for the host side; the third DataTable contains
+recommendations to users. Each DataTable is associated with some table
```

Comment:
While we're here, could we place these 3 points within a markdown list?

Reply:
Done